src/blocks/Transaction.vue

```diff
@@ -443,6 +442,12 @@ export default Vue.extend({
     if (!this.$store.getters["zk-account/loggedIn"]) {
       return;
     }
+    if ((this.mainToken === "L2-NFT" || this.type === "MintNFT") && this.$store.getters["zk-onboard/selectedWallet"] === "Argent") {
+      this.$accessor.openModal("ArgentNftWarning");
+      // @ts-ignore
+      this.$router.push(this.routeBack);
```

Comment:
why `@ts-ignore`? Use `await this.$router.push` or `return this.$router.push` to avoid warnings

Reply:
It's not the case how to fix it but I resolved the issue with typing